.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
```

Comment:
I don't know this, what will it change for non-windows contributors ?

Reply:
See this for an explaination https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_formatting_and_whitespace

I added this because when the pre-commit hook ran all of the files in the 'packages/mookme' directory failed because the line endings weren't converted on checkout. There are ways to fix this after checkout but I just overrode the hook because I only changed one file.

Reply:
A few decades of software development and yet it remains impossible to provide project specifics `.gitconfig` files :/

I'll trust you on this one as I can't really visualize what's at stake for non-windows users (but if I understand correctly, nothing ?)